.huskyrc.yml

```
@@ -0,0 +1,8 @@
+hooks:
+  commit-msg: commitlint -e
+  pre-commit: >
+    git stash -q --keep-index &&
+    npm run format &&
+    npm run build &&
+    git update-index --again &&
```

The command to stage the changes in the `lib/` directory should stay:

```suggestion
    git add lib/index.js &&
    git update-index --again &&
```